server/routes/api/results.js

```diff
@@ -0,0 +1,28 @@
+//results.js
+
+const express = require("express");
+const router = express.Router();
+
+//External Modules
+import { getVotes } from "../utils/getVotes";
+
+router.get("/results", (req, res) => {
+    if (req.body === null) {
+        res.send({ status: 400, error: "No poll id provided" });
+    } else {
+        try {
+            getVotes(req.body.pollId).then(results => {
+                res.send({ status: 200, data: results });
```

Comment:
As per comments above, could you please send results as `res.send(results)` or `res.status(200).send(results)`. Otherwise, I need to access results as `response.data.data` which is redundant. Thank you!